zookeeper-api/src/main/java/org/apache/helix/zookeeper/impl/factory/ZkConnectionManager.java

```diff
@@ -132,6 +131,10 @@ protected synchronized void close(boolean skipIfWatched) {
     LOG.info("ZkConnection {} was closed.", _monitorKey);
   }
 
+  protected boolean hasSharedWatchers() {
```

Comment:
Please change the other logic that doing the same check refers to this method.

Reply:
Thanks will take care as you mentioned in the below comment